vidgear/gears/asyncio/netgear_async.py

```diff
@@ -240,9 +244,13 @@ def __init__(
             # assign local ip address if None
             if address is None:
                 self.__address = "localhost"
+            else:
+                self.__address = address
             # assign default port address if None
             if port is None:
                 self.__port = "5555"
+            else:
+                self.__port = str(port)
```

Comment:
@otter-in-a-suit make `str(port)` => `port`. OR add str(port) above too. We need consistency in the code.